README.md

```diff
@@ -63,6 +63,10 @@ Flags:
       --links.validate.without-address-regex=^$  
```

Comment:
Let's kill this flag and use below one maybe?

(We can adjust flag names before first release - later it will be pain for users so let's make sure we got it right now)

Reply:
Sure! Would be easier!